README.md

```diff
@@ -10,6 +10,60 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple types in the same file, each type should have its own file.
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+Before the class declaration:
+
+* Enums (as private)
+
+Inside the class:
+
+* Public properties
+* Private properties
+* Public static variables
+* Private static variables
+* Init/Deinit
+* Override functions
+	* View lifecycle if in a view controller
+	* Other override functions 
+* Public functions
+* Private functions
+
+After the class declaration:
```

Comment:
The idea is we are separating out section ordering from general ordering; since `public/internal/private` applies to _all_ sections, we break it out. So you can simplify to what Paul posted above.